hyo2/soundspeed/formats/readers/sippican.py

```diff
@@ -162,6 +163,9 @@ def _parse_header(self):
                     self.ssp.cur.meta.sensor_type = Dicts.sensor_types['Unknown']
                     logger.warning("reverted to unknown probe type at line #%s" % self.samples_offset)
 
+            elif line[:len(self.tk_serial)] == self.tk_serial:
+                self.ssp.cur.meta.sn = line.split(':')[-1].lstrip().strip()
```

Comment:
Why do we need both [lstrip](https://docs.python.org/3/library/stdtypes.html?highlight=lstrip#str.lstrip) and [strip](https://docs.python.org/3/library/stdtypes.html?highlight=strip#str.strip) here?